azure-pipelines.yml

```diff
@@ -55,6 +55,76 @@ stages:
           - checkout: self
           - script: ./ci/kotlin_lint.sh
             displayName: 'Run Kotlin lint'
+  # Note: the size regression jobs are not ideal.
+  # The size analysis described in the docs was ran on arm64 machines.
+  # However, it seems that azure pipelines will support arm64 machines sometime soon.
+  # Whenever that is available this jobs should be moved to that architecture.
+  # Issue: https://github.com/lyft/envoy-mobile/issues/216
+  - stage: performance
+    dependsOn: [] # this removes the implicit dependency on previous stage and causes this to run in parallel.
+    jobs:
+      - job: size_current
+        timeoutInMinutes: 60
```

Comment:
Does the timeout apply to just while the job is running or does it include time in queue as well. If it's just the former, can this be shorter?

Reply:
Yep, this is only for running time. Yeah this can probably be lower. Looks like 30 mins is good enough.